packages/console/src/assets/docs/guides/web-remix/README.mdx

```diff
@@ -0,0 +1,187 @@
+  cookie: {
+    name: "logto-session",
+    maxAge: 14 * 24 * 60 * 60,
+    secrets: '${buildIdGenerator(12)()}', // Auto-generated secret
+  },
+});`}
+  </code>
+</pre>
+
+</Step>
+
+<Step
+  title="Init LogtoClient"
+  subtitle="1 step"
+>
+
+Use the `sessionStorage` created in the previous step to initialize LogtoClient:
+
+<pre>
+  <code className="language-ts">
+    {`// app/services/authentication.ts
+
+import { makeLogtoRemix } from "@logto/remix";
+
+export const logto = makeLogtoRemix(
+  {
+    endpoint: '${props.endpoint}',${
+      props.alternativeEndpoint ? ` // or "${props.alternativeEndpoint}"` : ''
+    }
+    appId: '${props.app.id}',
+    appSecret: '${props.app.secret}',
+    baseUrl: 'http://localhost:3000', // Change to your own base URL
+  },
+  { sessionStorage }
+);`}
+  </code>
+</pre>
+
+</Step>
+
+<Step
+  title="Mount auth routes"
+  subtitle="3 routes"
+>
+
+The SDK ships with a convenient function that mounts the authentication routes: sign-in, sign-in callback and the sign-out route.
+
+First, let’s enter your redirect URI. E.g. `http://localhost:3000/api/logto/sign-in-callback`.
+
+<UriInputField name="redirectUris" />
+
+Then mount 3 routes in your app:
+
+```ts
+// app/routes/api/logto/$action.ts
+
+import { logto } from "../../../services/authentication";
+
+export const loader = logto.handleAuthRoutes({
+  "sign-in": {
+    path: "/api/logto/sign-in",
+    redirectBackTo: "/api/logto/callback",
+  },
+  "sign-in-callback": {
+    path: "/api/logto/callback",
+    redirectBackTo: "/",
+  },
+  "sign-out": {
+    path: "/api/logto/sign-out",
+    redirectBackTo: "/",
+  },
+});
+```
+
+As you can see, the mount process is configurable and you can adjust it for your particular route structure. The whole URL path structure can be customized via the passed configuration object.
+
+When mounting the routes as described above, you can navigate your browser to `/api/logto/sign-in` and you should be redirected to your Logto instance where you have to authenticate then.
+
+</Step>
+
+<Step
+  title="Get user profile"
+  subtitle="and authentication context"
+>
+
+A typical use case is to fetch the _authentication context_ which contains information about the respective user. With that information, it is possible to decide if the user is authenticated or not. The SDK exposes a function that can be used in a Remix `loader` function:
+
+```ts
+// app/routes/index.tsx
+import type { LogtoContext } from "@logto/remix";
+import { LoaderFunction, json } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+
+import { logto } from "~/services/authentication";
+
+type LoaderResponse = {
+  readonly context: LogtoContext;
+};
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const context = await logto.getContext({ getAccessToken: false })(
+    request
+  );
+
+  if (!context.isAuthenticated) {
+    return redirect("/api/logto/sign-in");
```

Comment:
same here should we use the redirect uri path ?

Reply:
No, this is expected, redirect to the Remix app to trigger sign in.